tests/garage/torch/algos/test_vpg.py

```diff
@@ -0,0 +1,91 @@
+import gym
+import pytest
+import torch
+
+from garage.envs.base import GarageEnv
+from garage.experiment import deterministic
+from garage.experiment import LocalRunner
+from garage.np.baselines import LinearFeatureBaseline
+from garage.torch.algos import VPG
+from garage.torch.policies import GaussianMLPPolicy
+from tests.fixtures import snapshot_config
+
+# yapf: disable
+INVALID_ENTROPY_CONFIG = [
+    ({'entropy_method': 'INVALID_ENTROPY_METHOD'},
+        ValueError, 'entropy_method'),
+    ({'entropy_method': 'max',
+      'center_adv': True},
+        ValueError, 'center_adv'),
+    ({'entropy_method': 'max',
+      'center_adv': False,
+      'stop_entropy_gradient': False},
+        ValueError, 'entropy_method'),
+    ({'entropy_method': 'no_entropy',
+      'policy_ent_coeff': 1.0},
+        ValueError, 'policy_ent_coeff')
+]
```

Comment:
i think replacing this with `]  # yapf: disable` will achieve the affect you were going for.

Reply:
add `# yapf: enable` on the bottom